translationCoreApps/tC_Resources
# Feature/3104/Convert BHP to chapter files.

## This PR Addresses

Adding a helper to convert a usfm file from path to json chapter files given a path.
Also added a script to download BHP files and convert to chapter files: `./script/bhpParse.js`.  See instructions to run in that file.  Currently the format of the bhp usfm files are being changed from using key `strongs` to using `strong`.  So do not generate and commit new bhp files until tC has been updated.

## To Test this PR

- Run `npm install` inside `tC_resources` path to install node_modules
- Run `npm test` to verify that the tests pass to ensure functionality


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/tc_resources/12)
